prepare/cards/20newsgroups_sklearn.py

```diff
@@ -42,13 +42,7 @@
         ),
         RenameFields(field_to_field={"data": "text", "target": "label"}),
         MapInstanceValues(mappers={"label": map_labels}),
-        Set(
-            fields={
-                "classes": list(map_labels.values()),
-                "text_type": "text",
-                "type_of_class": "topic",
```

Comment:
While topic classification is a common task, it can not be assumed as a default for "tasks.classification.multi_class", because many other (e.g .sentiment) would get a wrong default.
 Thus I would recommend removing "type_of_class": "topic" from the  "tasks.classification.multi_class" and create a new "tasks.classification.multi_class.topic_classification" which that default.